data/docs/pachyderm-beginner-tutorial.yaml

```
@@ -0,0 +1,14 @@
+metadata:
+  name: pachyderm-beginner-tutorial
+  type: tutorial
+  annotations:
+    opendatahub.io/categories: 'AI/Machine learning, Pipeline'
+spec:
+  appName: pachyderm
+  displayName: Learn Pachyderm main concepts
```

```suggestion
  displayName: Getting started with Pachyderm concepts
``` 
Starting the title with a gerund to align with content consistency standards. Changed to "Getting started" to let users know that this tutorial is aimed towards beginners who are getting familiar with Pachyderm concepts.

---

@nadegepepin Could you please review?